Networking/Networking/Remote/RefundsRemote.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+import Alamofire
+
+/// Refunds: Remote Endpoints
+///
+public class RefundsRemote: Remote {
+
+    /// Retrieves a specific list of `OrderRefunds`s by `orderID`.
+    ///
+    /// - Parameters:
+    ///     - siteID: Site for which we'll fetch remote orders.
+    ///     - orderID: Unique identifier for the resource "order" which you are searching
+    ///                 for.
+    ///     - context: view or edit. Scope under which the request is made;
+    ///                determines fields present in response. Default is view.
+    ///     - pageNumber: Number of page that should be retrieved.
+    ///     - pageSize: Number of Orders to be retrieved per page.
+    ///     - completion: Closure to be executed upon completion.
+    ///
+    public func loadOrderRefunds(for siteID: Int,
+                                 by orderID: Int,
+                                 context: String = Default.context,
+                                 pageNumber: Int = Default.pageNumber,
+                                 pageSize: Int = Default.pageSize,
+                                 completion: @escaping ([OrderRefund]?, Error?) -> Void) {
+        let parameters = [
+            ParameterKey.page: String(pageNumber),
+            ParameterKey.perPage: String(pageSize),
+            ParameterKey.contextKey: context
+        ]
+        let path = String(format: Path.orderRefunds, orderID)
```

Comment:
I understand that you are trying to declare the path in just one constant, but we tend to avoid constructing the path with formatted strings, and either interpolate or just concatenate the path. 

Here is one example in a different service: https://github.com/woocommerce/woocommerce-ios/blob/1e01ae8a93d7245c42396e4e10ae715a91ee607a/Networking/Networking/Remote/CommentRemote.swift#L64

Reply:
I fixed it! 👍I took inspiration from `DevicesRemote` class, that use the same approach. Should it also be corrected there, or is it better to leave it out of this PR?
        
https://github.com/woocommerce/woocommerce-ios/blob/1e01ae8a93d7245c42396e4e10ae715a91ee607a/Networking/Networking/Remote/DevicesRemote.swift#L76

Reply:
Nice catch! Let's leave it out of this PR, to keep the diff focused.